### ISSUE TITLE
extension-installer doesn't support this package

### ISSUE THREAD

**Issue by Toilal** [mavimo/phpstan-junit#20]
```
phpstan/extension-installer: Extensions installed
> phpstan/phpstan-doctrine: installed
> phpstan/phpstan-phpunit: installed
> phpstan/phpstan-symfony: installed
> mavimo/phpstan-junit: not supported
```

Then I have an error when I try to use the junit formatter

```
$ vendor/bin/phpstan analyze --level 7 --no-progress --no-interaction --error-format=junit src tests
Note: Using configuration file /var/www/html/backend/phpstan.neon.
File '/var/www/html/backend/vendor/mavimo/phpstan-junit/phpstan.neon' is missing or is not readable.
```

I still have to include the following configuration

```
includes:
    - vendor/mavimo/phpstan-junit/extension.neon
```

**Comment by mavimo** [comment]
@Toilal thx for the feedback, I'm going to check it ASAP

**Comment by mavimo** [comment]
@Toilal should be fixed in v0.2.1, let me know if you find some issue on usage (I add integration test that check the status, but i'm not sure to I have no forget some check)

**Comment by Toilal** [comment]
I just give it a try, and it works, thanks !
